### PR TITLE
NSA 5308 overlap text in support

### PR DESCRIPTION
**Pull request by MarekArtur**
Wrapping long username of the support project.

### CODE REVIEW

File: src/app/users/views/summaryPartial.ejs [hunk 1]
```diff
@@ -1,128 +1,129 @@
...
         </div>
+        <% } %>
+    <% if(locals.user.pendingEmail) { %>
+    <div class="notification notification-information" tabindex="0">
+        <h2>Pending email address change</h2>
+        <p>Waiting for <%= user.name %> to confirm the new email address of <%= user.pendingEmail %>.</p>
+        <p>
+        <form method="post" action="cancel-change-email">
+            <input type="hidden" name="_csrf" value="<%= csrfToken %>"/>
+            <button class="button-text">Cancel change email</button>
```
Comment (AMcCartneyMTH) [inline src/app/users/views/summaryPartial.ejs]: Can we change this to just "Cancel"

File: src/app/users/views/summaryPartial.ejs [hunk 2]
```diff
@@ -1,128 +1,129 @@
...
+                        <a href="confirm-invitation-reactivation" class="action">Reactivate invitation</a>
+                    </p>
+                </div>
+            </div>
         </div>
+        <% } %>
+    <% if(locals.user.pendingEmail) { %>
+    <div class="notification notification-information" tabindex="0">
+        <h2>Pending email address change</h2>
+        <p>Waiting for <%= user.name %> to confirm the new email address of <%= user.pendingEmail %>.</p>
```
Comment (AMcCartneyMTH) [inline src/app/users/views/summaryPartial.ejs]: Can we change this to:

<p>Awaiting confirmation of email address change to <%= user.pendingEmail %>.</p>

File: src/app/users/views/summaryPartial.ejs [hunk 3]
```diff
@@ -1,128 +1,129 @@
...
-<div class="row">
-    <div class="col-12">
-        <div class="notification notification-success">
-            <%= locals.flash.info %>
+    <% if(locals.user.status.id === -2) { %>
+        <div class="row">
+            <div class="col-12">
+                <div class="notification">
+                    <p>
+                        This account invitation was deactivated
```
Comment (AMcCartneyMTH) [inline src/app/users/views/summaryPartial.ejs]: Can this be changed to:

<%= locals.flash.info %>

File: src/app/users/views/summaryPartial.ejs [hunk 4]
```diff
@@ -1,128 +1,129 @@
...
+                        <% if(locals.user.status.changedOn) { %>
+                        on <%= locals.moment(locals.user.status.changedOn).format('DD MMM YYYY') %>.
+                        <% } %>
+                        <a href="confirm-invitation-reactivation" class="action">Reactivate invitation</a>
```
Comment (AMcCartneyMTH) [inline src/app/users/views/summaryPartial.ejs]: Can we change this to:

<a href="confirm-invitation-reactivation" class="action">Reactivate</a>

File: src/app/users/views/summaryPartial.ejs [hunk 5]
```diff
@@ -1,128 +1,129 @@
...
-            <aside>
-                <h2 class="heading-medium">Actions</h2>
-                <ul class="list">
-                    <li><a href="associate-organisation">Add organisation</a></li>
-                    <% if (locals.organisations.length > 0) { %>
-                        <li><a href="select-organisation">Add services</a></li>
-                    <% } %>
-                    <li><a href="edit-email">Change user's email address</a></li>
-                    <li><a href="confirm-invitation-deactivation">Deactivate user</a></li>
-                    <li><a href="resend-invitation"> Resend invitation email</a></li>
```
Comment (AMcCartneyMTH) [inline src/app/users/views/summaryPartial.ejs]: Not sure why this is being replaced when we are adding a 'text-break' class to a h1 element?

File: src/app/users/views/summaryPartial.ejs [hunk 6]
```diff
@@ -1,128 +1,129 @@
...
-        <% if(!locals.isInvitation) { %>
-        <% if(locals.area === 'audit') { %>
-        <li class="active">Audit</li>
-        <% } else { %>
-        <li><a href="audit" role="tab">Audit</a></li>
-        <% } %>
-        <% } %>
-
-    </ul>
-</nav>
```
Comment (AMcCartneyMTH) [inline src/app/users/views/summaryPartial.ejs]: Not sure why this is being replaced when the scope of the ticket is to enforce the wrapping of text. Could you expand on this, please?

**Review by AMcCartneyMTH — CHANGES_REQUESTED** [review]
Comments have been added in order to clarify the changes being proposed.

File: src/app/users/views/summaryPartial.ejs (hunk 4 above)
Comment (MarekArtur) [reply src/app/users/views/summaryPartial.ejs]: Done

File: src/app/users/views/summaryPartial.ejs (hunk 5 above)
Comment (MarekArtur) [reply src/app/users/views/summaryPartial.ejs]: Paragraph formatting.

File: src/app/users/views/summaryPartial.ejs (hunk 6 above)
Comment (MarekArtur) [reply src/app/users/views/summaryPartial.ejs]: Paragraph formatting.

File: src/app/users/views/summaryPartial.ejs (hunk 4 above)
Comment (MarekArtur) [reply src/app/users/views/summaryPartial.ejs]: Done

File: src/app/users/views/summaryPartial.ejs (hunk 3 above)
Comment (MarekArtur) [reply src/app/users/views/summaryPartial.ejs]: DOne

File: src/app/users/views/summaryPartial.ejs (hunk 2 above)
Comment (MarekArtur) [reply src/app/users/views/summaryPartial.ejs]: Done

File: src/app/users/views/summaryPartial.ejs (hunk 1 above)
Comment (MarekArtur) [reply src/app/users/views/summaryPartial.ejs]: Done

Comment (MarekArtur) [reply src/app/users/views/summaryPartial.ejs]: Done

File: src/app/users/views/summaryPartial.ejs (hunk 3 above)
Comment (MarekArtur) [reply src/app/users/views/summaryPartial.ejs]: Done

**Review by AMcCartneyMTH — APPROVED** [review]
Code reviewed.

**Comment by AMcCartneyMTH** [comment]
Rolling back changes to correct build error.

**Pull request merged**